jenkins-pipelines/oss/artifacts/artifacts-docker.jenkinsfile

```diff
@@ -9,4 +9,6 @@ artifactsPipeline(
 
     timeout: [time: 30, unit: 'MINUTES'],
     post_behavior_db_nodes: 'destroy'
+
+    extra_environment_variables: 'SCT_USE_MGMT=false'
```

Comment:
This patch is not working :-( , you are missing `,` after `post_behavior_db_nodes: 'destroy'`

```
04:53:29  WorkflowScript: 13: expecting ')', found 'extra_environment_variables' @ line 13, column 5.
04:53:29         extra_environment_variables: 'SCT_USE_MGMT=false'
04:53:29         ^
04:53:29  
04:53:29  1 error
```